cmd/controller.go

```diff
@@ -50,6 +51,7 @@ func init() {
 	harborAPIEndpoint = flags.String("harbor-api-endpoint", "", "URL to the Harbor API Endpoint")
 	harborAPIUsername = flags.String("harbor-username", "", "Harbor username to use for authentication")
 	harborAPIPassword = flags.String("harbor-password", "", "Harbor password to use for authentication")
+	skipVerifyTls = flags.Bool("skip-tls-verification", false, "Skip TLS certificate verification")
```

Comment:
Could you please add this flag to the [config reference docs](https://github.com/moolen/harbor-sync/blob/master/docs_src/content/docs/configuration.md) and [regenerate](https://moolen.github.io/harbor-sync/docs/development/#documentation) the docs?

Apart from that: LGTM :+1: :heart:

Reply:
Done!, should be fine now